.github/workflows/ci.yml

```diff
@@ -75,6 +75,15 @@ jobs:
           key: ${{ runner.os }}-ccache-${{ steps.timestamp.outputs.timestamp }}
           restore-keys: ${{ runner.os }}-ccache-
 
+      # Cache chocolatey packages on Windows.
+      - name: Cache Chocolatey Dependencies
```

Comment:
Nit: I'd like the name of the steps to converge on a 'casing'. I'd prefer non-title case, but I mostly want consistency.